policy-controller/k8s/index/Cargo.toml

```diff
@@ -11,6 +11,7 @@ anyhow = "1"
 futures = { version = "0.3", default-features = false }
 http = "0.2"
 k8s-gateway-api = "0.15"
+k8s-openapi = { version = "0.20", features = ["schemars"] }
```

Comment:
I think this should go down under the `[dev-dependencies]` section given it's only used for testing.